farmsharescripts/helpers_farmshare.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+#
```

Comment:
Why did you replicate all of the scripts instead of discussing or thinking about how to maintain or update them? This will be hugely challenging to maintain…

Reply:
Ah sorry! I though it would be easier to have the farmshare stuff separate from the sherlock stuff. No worries, I will refactor them into a single script.

Reply:
No worries! We can definitely still have a farmshare folder of batch scripts, and possibly other files if they don't fit into the current framework, but duplicating the entire code as I mentioned about will be very hard to maintain. If it's possible to use the same scripts for farmshare (which I believe it is) this would be the better way to go. I 100% support an extra page of documentation just for farmshare if anything is different.